packages/web/src/features/users/ResetPasswordView.tsx

```diff
@@ -0,0 +1,79 @@
+import ModalView, { ModalViewTitle } from '../../shared/components/ModalView';
+import FormLabel from '../../shared/components/form/FormLabel';
+import InputError from '../../shared/components/form/InputError';
+import SubmitButton from '../../shared/components/form/SubmitButton';
+import TextInput from '../../shared/components/form/TextInput';
+import { useForm } from 'react-hook-form';
+import { useFlash } from '../../shared/hooks/flash';
+import apiClient from '../../shared/apiClient';
+import { ApiClientError } from '@translation/api-client';
+import Form from '../../shared/components/form/Form';
+import { useTranslation } from 'react-i18next';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import useAuth from '../../shared/hooks/useAuth';
+import useTitle from '../../shared/hooks/useTitle';
+
+export default function ResetPasswordView() {
+  const { t } = useTranslation(['users', 'common']);
+  useTitle(t('common:tab_titles.reset_password'));
```

Comment:
```suggestion
  useTitle(t('users:reset_password'));
```

I would remove the question mark from `users:reset_password` and use it here. No need to have duplicate terms.

Reply:
Done :+1: Question mark removed from `users:forgot_password`